tldw_Server_API/app/core/Ingestion_Media_Processing/MediaWiki/Media_Wiki.py

```
@@ -173,34 +176,34 @@ def validate_file_path(file_path: str, allowed_dir: Optional[Path] = None) -> Pa
         # Check if it's a file (not a directory or symlink to directory)
         if not path.is_file():
-            raise ValueError("Path is not a regular file")
+            raise InvalidStoragePathError("Path is not a regular file")
 
         # Check for symlink attacks
         if path.is_symlink():
             # Resolve the symlink and check if it's within allowed directory
             real_path = path.resolve()
             if allowed_dir and not str(real_path).startswith(str(allowed_dir)):
-                raise ValueError("Symlink points outside allowed directory")
+                raise InvalidStoragePathError("Symlink points outside allowed directory")
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)